### PR TITLE
fix(rpc): allow mobile app to communicate with webview

### PR DESCRIPTION
**Pull request by DeekyJay**

### CODE REVIEW

File: src/bundle.ts [hunk 1]
```diff
@@ -13,7 +13,14 @@ export * from './typings';
 export * from './decoration';
 export * from './package';
 
-const rpc = new RPC(window.top, '1.0');
+const rpc = new RPC(
+  window.top,
+  '1.0',
+  '*',
+  undefined,
```
Comment (ProbablePrime) [inline src/bundle.ts]: If these are undefined here can we move the arguments on the constructor so that they can be omitted?

Comment (ProbablePrime) [reply src/bundle.ts]:
```

Comment (ProbablePrime) [reply src/bundle.ts]:
```

Comment (ProbablePrime) [reply src/bundle.ts]:
```
 private readonly target: IPostable,
    protocolVersion: string,	    protocolVersion: string,
    private readonly origin: string = '*',	    private readonly origin: string = '*',
private readonly isApp?: boolean,
    private readonly setOnMessage?: (fn: (ev: any) => void) => void,
    private readonly removeOnMessage?: (fn: (ev: any) => void) => void,
    
```

Would then mean we wouldn't have to include them?

File: src/rpc.ts [hunk 2]
```diff
@@ -82,6 +82,8 @@ export class RPC extends EventEmitter {
   private remoteCallQueue: RPCMessageWithCounter<any>[] = [];
   private lastSequentialCall = -1;
   private remoteProtocolVersion: string | undefined;
+  private handler: Window | Document = window;
+  private dataIsObject: boolean = false;
```
Comment (ProbablePrime) [inline src/rpc.ts]: Does this need to be a member variable?

File: src/bundle.ts (hunk 1 above)
Comment (DeekyJay) [reply src/bundle.ts]: I've reordered these so they can be omitted 😄

File: src/rpc.ts (hunk 2 above)
Comment (DeekyJay) [reply src/rpc.ts]: dataIsObject just keeps reference on whether or not we should JSON.parse or not

**Pull request merged**